content/spin/v2/variables.md

```diff
@@ -117,42 +117,37 @@ fn handle_spin_example(req: Request) -> Result<impl IntoResponse> {
 
 {{ startTab "TypeScript"}}
 
-> [**Want to go straight to the reference documentation?**  Find it here.](https://fermyon.github.io/spin-js-sdk/variables/Config.html)
+> [**Want to go straight to the reference documentation?**  Find it here.](https://fermyon.github.io/spin-python-sdk/variables.html)
 
-The function is available on [the `Config` object](https://fermyon.github.io/spin-js-sdk/variables/Config.html) and is named [`get`](https://fermyon.github.io/spin-js-sdk/interfaces/_internal_.SpinConfig.html#get).
-
-> Note that the name is `Config` rather than `Variables`.
+The `variables` module has a function called `get`(https://fermyon.github.io/spin-python-sdk/variables.get).
 
 ```ts
-import { HandleRequest, HttpRequest, HttpResponse, Config } from "@fermyon/spin-sdk"
-
-const decoder = new TextDecoder("utf-8")
-
-export const handleRequest: HandleRequest = async function (request: HttpRequest): Promise<HttpResponse> {
-  const expected = decoder.decode(request.body)
-  let password = Config.get("password")
-  let access = "denied"
-  if (expected === password) {
-      access = "accepted"
-  }
-  let responseJson = `{\"authentication\": \"${access}\"}`;
-
-  return {
-    status: 200,
-    headers: { "Content-Type": "application/json" },
-    body: responseJson
-  }
-}
-
+from spin_sdk import http
+from spin_sdk.http import  Request, Response
+from spin_sdk import variables
+
+class IncomingHandler(http.IncomingHandler):
+    def handle_request(self, request: Request) -> Response:
+        password = request.body.decode("utf-8")
+        expected = variables.get("password")
+        access = "denied"
+        if expected == password:
+            access = "accepted"
+        response = f'\{{"authentication": "{access}"}}'
+        return Response(
+            200,
+            {"content-type": "text/plain"},
+            bytes(response, "utf-8")
+        )
 ```
 
 {{ blockEnd }}
 
 {{ startTab "Python"}}
 
-> [**Want to go straight to the reference documentation?**  Find it here.](https://fermyon.github.io/spin-python-sdk/v1/spin_config.html)
+> [**Want to go straight to the reference documentation?**  Find it here.](https://fermyon.github.io/spin-python-sdk/variables.html)
 
-The config function is available in the `spin_config` package and is named [`config_get`](https://fermyon.github.io/spin-python-sdk/v1/spin_config.html#spin_sdk.spin_config.config_get).
+The config function is available in the `spin_config` package and is named [`config_get`](https://fermyon.github.io/spin-python-sdk/variables.html#spin_sdk.spin_config.config_get).
```

Comment:
I thought we got rid of the `config` terminology?  The ``variables` page in the docs exists but is empty so I'm not sure what's going on.

Reply:
Looks like the docs generator does not generate docs for re-exports. 
<img width="920" alt="image" src="https://github.com/fermyon/developer/assets/25023490/22148b5e-fe85-4134-9249-a7ad5eb1cf7d">

I can fix that in the docs.